### PR TITLE
add get request into github api

### PR DESCRIPTION
**Pull request by aperdomob**

### CODE REVIEW

File: test/GithubApi.Repositories.test.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const { expect, assert } = require('chai');
+const md5 = require('md5');
+
+const urlBase = 'https://api.github.com';
+
+describe('Given a user logged in github', () => {
+  const username = 'aperdomob';
```
Comment (luigisamurai) [inline test/GithubApi.Repositories.test.js]: You use two users in the tests.  does it work with the same token?

File: test/GithubApi.Repositories.test.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+  describe(`when get ${username} user`, () => {
+    let user;
+
+    before(() => {
+      const query = agent.get(`${urlBase}/users/${username}`)
```
Comment (luigisamurai) [inline test/GithubApi.Repositories.test.js]: you should improve the name of this variable

File: test/GithubApi.Repositories.test.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      expect(user.location).to.equal('Colombia');
+    });
+
+    describe('when get his repos', () => {
+      let repositories;
+      let repository;
+      const expectedRepository = 'jasmine-awesome-report';
+
+      before(() => {
+        const query = agent.get(user.repos_url)
```
Comment (luigisamurai) [inline test/GithubApi.Repositories.test.js]: the same comment that above

File: test/GithubApi.Repositories.test.js (hunk 1 above)
Comment (aperdomob) [reply test/GithubApi.Repositories.test.js]: we need only logged one time

**Pull request merged**